pkg/cli/cmd/utils.go

```diff
@@ -94,3 +97,20 @@ func CheckIfRecipeExists(ctx context.Context, client clients.ApplicationsManagem
 
 	return envResource, recipeProperties, nil
 }
+
+// GetConnection from Workspace.
+func GetConnection(ctx context.Context, workspace *workspaces.Workspace) (sdk.Connection, error) {
+	connection, err := workspace.Connect()
+	if err != nil {
+		return nil, err
+	}
+
+	err = sdk.TestConnection(ctx, connection)
+	if errors.Is(err, &sdk.ErrRadiusNotInstalled{}) {
+		return nil, clierrors.MessageWithCause(err, "Could not connect to Radius.")
+	} else if err != nil {
+		return nil, err
+	}
+
+	return connection, nil
```

Comment:
Is this duplicating some existing code? Can we can consolidate it on a existing entity. 

Should we make a `workspace.ConnectAndTest()` for example?